syntax = "proto2";

package fuzz_topic_matches_sub;

message FuzzerInput {
	required string string1 = 1;
	required string string2 = 2;
	optional string username = 3;
	optional string clientid = 4;
}
